From c2818c508290acc2144beb33828cc5496c4ecef2 Mon Sep 17 00:00:00 2001
From: Stefan Lellwitz <lel@iesy.com>
Date: Tue, 19 Jul 2022 15:14:04 +0200
Subject: [PATCH 1/1] arm64: dts: rockchip: px30: set drive strength for i2c0
 to 8mA

This resolves i2c bus conflict. Console messages won't be destroyed anymore when using mikroBUS modules.

Signed-off-by: Stefan Lellwitz <lel@iesy.com>
---
 arch/arm/dts/px30-evb.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/dts/px30-evb.dts b/arch/arm/dts/px30-evb.dts
index edf9b0b2dd7e..a2f93a4ab1d1 100644
--- a/arch/arm/dts/px30-evb.dts
+++ b/arch/arm/dts/px30-evb.dts
@@ -439,6 +439,14 @@
 			<0 7 RK_FUNC_GPIO &pcfg_pull_up>;
 		};
 	};
+
+	i2c0 {
+	i2c0_xfer: i2c0-xfer {
+		rockchip,pins =
+			<0 RK_PB0 RK_FUNC_1 &pcfg_pull_none_8ma>,
+			<0 RK_PB1 RK_FUNC_1 &pcfg_pull_none_8ma>;
+		};
+	};
 };
 
 &u2phy {
-- 
2.30.2

